finetune/sequence_labeling.py

```diff
@@ -72,43 +98,27 @@ def predict(self, X):
         :param X: A list / array of text, shape [batch]
         :returns: list of class labels.
         """
-        subseqs, _ = indico_to_finetune_sequence(X, multi_label=self.multi_label)
-
         chunk_size = self.config.max_length - 2
         step_size = chunk_size // 3
-        
-        arr_encoded = self._text_to_ids(subseqs)
-
-        labels = []
-        batch_probas = []
-        with warnings.catch_warnings():
-            warnings.filterwarnings("ignore")
-            for xmb, mmb in self._infer_prep(subseqs):
-                output = self._eval(self.predict_op,
-                    feed_dict={
-                        self.X: xmb,
-                        self.M: mmb,
-                        self.do_dropout: DROPOUT_OFF
-                    }
-                )
-                prediction, probas = output.get(self.predict_op)
-                batch_probas.extend(probas)
-                formatted_predictions = self.label_encoder.inverse_transform(prediction)
-                labels.extend(formatted_predictions)
+        arr_encoded = list(itertools.chain.from_iterable(self.input_pipeline._text_to_ids([x]) for x in X))
```

Comment:
We should use a different variable name now that we've changed the data format from ArrayEncodedOutput here.